specification/window.browser.bs

```diff
@@ -52,6 +52,39 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
     it MUST be used only for WebExtension related functionality.
   </section>
 
+  <section>
+    For backwards compatibility with older versions of web extensions, a UA MAY
```

Comment:
Should we replace web extensions with WebExtensions to be consistent with the other usages in this document?

Reply:
yep!